src/theme/components/heading.ts

```diff
@@ -172,7 +172,7 @@ export const headings = {
       },
       fontWeight: "heading.heading8",
       letterSpacing: "0",
-      lineHeight: "1.50",
+      lineHeight: { base: "1.50", md: "1.50" }, // This value is intensionally the same for mobile and desktop.
```

Comment:
You should be able to just set this to `lineHeight: "1.50"` and it should be set for both mobile and the medium breakpoint.

As an aside, is `1.50` not the same as `1.5`?

Reply:
Right. I will simplify and then update my comment to explain why this one is not setup to be responsive like the others.

Reply:
Also, can you share documentation about the difference of `1.50` and `1.5` for `line-height`?

Reply:
I don't know if it's true that those two values `1.50` and `1.5` are not the same so I was asking, but I can look for resources.

Reply:
Sorry, I read that as a statement and not as a question. Yes, `1.50` and `1.5` are numerically the same. However, the designer in me opted to use the same format as those that do require two decimal places.  Symmetry. Consistency. All that.